engine/install/linux-postinstall.md

```diff
@@ -151,7 +151,7 @@ the [Docker CLI Reference](/engine/reference/commandline/dockerd/) article.
 > understand the security implications of opening docker to the network. If steps are not taken to secure the connection, 
 > it is possible for remote non-root users to gain root access on the host. For more information on how to use TLS 
 > certificates to secure this connection, check this article on 
-> [how to protect the Docker daemon socket](../security/https.md).
+> [how to protect the Docker daemon socket](../security/protect-socket.md).
```

Comment:
Thinking out loud; would `secure-api`, `protect-access` or something along those lines be more descriptive? (as it's not necessarily the "socket" we're protecting, but protecting access to the engine API.

Perhaps we should move this warning to an include-file, so that we can more easily re-use it (thinking of adding the same / similar warning on https://docs.docker.com/engine/api/)

Reply:
Changed to protect-access